src/test/java/org/folio/rest/api/AuditDeleteTest.java

```diff
@@ -127,26 +130,40 @@ private Object getRecordIdFromAuditTable(String tableName)
 
     final JsonArray objects = getRecordsFromAuditTable(tableName).get(0);
     final JsonPointer jsonPointer = JsonPointer.from(RECORD_ID_JSON_PATH);
-    return jsonPointer.queryJson(new JsonObject(objects.getString(1)));
+    return jsonPointer.queryJson((JsonObject) objects.getValue(1));
   }
 
   private List<JsonArray> getRecordsFromAuditTable(String tableName)
     throws InterruptedException, TimeoutException, ExecutionException {
 
     final CompletableFuture<List<JsonArray>> result = new CompletableFuture<>();
     postgresClient.select(getAuditSQL(tableName), h -> {
-      result.complete(h.result().getResults());
+      RowIterator<Row> iterator = h.result().iterator();
+      List<JsonArray> list = new LinkedList<>();
+      while (iterator.hasNext()) {
+        Row row = iterator.next();
+        JsonArray ar = new JsonArray();
+        for (int i = 0; i < row.size(); i++) {
+          Object obj = row.getValue(i);
+          if (obj instanceof java.util.UUID) {
+            ar.add(obj.toString());
+          } else {
+            ar.add(obj);
+          }
+        }
+        list.add(ar);
+      }
+      result.complete(list);
```

Comment:
Could this be extracted out into a separate method, both to explain what the intent is and to reduce the noise in the test?

Reply:
Yep. Will do

Reply:
Done